src/pages/index.js

```diff
@@ -19,6 +20,26 @@ class Home extends Component {
         })
     }
 
+    handleEscape = e => {
```

Comment:
I think the keycodes should be handled in the modal component? you can pass down a function that sets `show_modal` to false